app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -122,6 +124,34 @@ const MeetingEditor = () => {
       : undefined,
   );
 
+  //For some reason calling the debounce function directly was wack.
+  //Also, we want to check if the new length is greater than the old one or if it doesn't exist already.
+  const updateReportValue = useCallback(
+    (newReportValue) => {
+      const debounceFunc = debounce(() => {
+        const storedReport = sessionStorage.getItem(
+          `meeting-${meetingId}-report`,
+        );
+        if (!storedReport || storedReport.length < newReportValue.length) {
```

Comment:
Thoughts on doing this based on a timestamp rather than the length of the report? Even though this probably works in the majority of the cases, a longer report doesn't always mean a newer one